cylc/flow/scripts/set_verbosity.py

```diff
@@ -52,30 +54,36 @@ def get_option_parser():
     parser = COP(
         __doc__, comms=True,
         argdoc=[
-            ('WORKFLOW', 'Workflow name or ID'),
-            ('LEVEL', ', '.join(LOG_LEVELS.keys()))
+            ('LEVEL', ', '.join(LOG_LEVELS.keys())),
+            ('ID [ID ...]', 'Workflow ID(s)'),
```

Comment:
Are we sure about reversing the order of these args?

Reply:
Yep, workflow args must always come last (because we don't know how many of them there will be).